ecl/eclagent/eclagent.cpp

```diff
@@ -2016,8 +2016,9 @@ void EclAgent::doProcess()
         {
             if (w->getDebugValueBool("analyzeWorkunit", agentTopology->getPropBool("@analyzeWorkunit", true)))
             {
+                double costPerMs = calculateThorCost(1, getNodes());
```

Comment:
detail: Different graphs could have been executed on different width thors (if targetted to a different cluster).  However that can be a detail to revisit.

Reply:
I've created a Jira to look into this later: https://track.hpccsystems.com/browse/HPCC-27265